toolkit/templates/default-custom-dimensions.html

```diff
@@ -0,0 +1,14 @@
+{# User-level custom dimensions #}
+{% if current_user.role %}
+  <meta name="ga_customDimension" data-id="10" data-value="{{ current_user.role }}">
+{% endif %}
+
+{# User organisation size custom dimention #}
+{% if current_user and current_user.supplier_organisation_size %}
+  <meta name="ga_customDimension" data-id="11" data-value="{{ current_user.supplier_organisation_size }}">
+{% endif %}
+
+{# Custom dimention for user research banner #}
+{% if not current_user.user_research_opted_in and not 'seen_user_research_message' in request.cookies %}
+  <meta name="ga_customDimension" data-id="12" data-value="User research banner">
+{% endif %}
```

Comment:
Let's chat about this - I think it's getting overcomplicated, given that we are only using `custom_meta` in one place (Buyer FE `templates/direct-award/view-project.html`).  We definitely don't need two templates!